src/components/Idler.js

```diff
@@ -40,7 +40,7 @@ class Idler extends PureComponent<Props> {
 
   lastAction: number = -1
 
-  debounceOnChange = debounce(_ => this.idleTimeHandler(), 1000)
+  debounceOnChange = debounce(_ => this.idleTimeHandler(), 1000, { maxWait: 1000, leading: true })
```

Comment:
as `idleTimeHandler` is automatically bound with `this` because of arrow, you can directly

```
debounce(this.idleTimeHandler, 1000, { maxWait: 1000, leading: true })
```

:100: for the `maxWait`

Reply:
Oh nice catch